pkg/crc/config/viper_config.go

```diff
@@ -37,6 +37,7 @@ func NewViperStorage(configFile, envPrefix string) (*ViperStorage, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error reading configuration file '%s': %v", configFile, err)
 	}
+	v.WatchConfig()
```

Comment:
Why don't you make this change as part of https://github.com/code-ready/crc/pull/1558/commits/3a15992263c6aad9c922031dea22505b7935c371 commit itself? I mean it doesn't make sense you first remove the Watchconfig and then add it back?

Reply:
Will do, i removed it then the test failed so added it again!